go.mod

```diff
@@ -72,17 +59,33 @@ require (
 	github.com/uptrace/opentelemetry-go-extra/otelutil v0.1.16 // indirect
 	go.etcd.io/etcd/client/pkg/v3 v3.5.4 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/internal/retry v1.10.0 // indirect
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.9.0 // indirect
 	go.opentelemetry.io/proto/otlp v0.19.0 // indirect
+	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3 // indirect
+	golang.org/x/text v0.3.7 // indirect
+	golang.org/x/xerrors v0.0.0-20220609144429-65e65417b02f // indirect
+	google.golang.org/protobuf v1.28.1 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+require (
```

Comment:
nit: this file was weirdo even before this PR: three `require` groups. Now it has **four** `require` groups...
when there should be only two groups: one for direct deps, and another for indirect:

> indirect dependencies are recorded in a separate block within the go.mod file.

https://go.dev/ref/mod#go-mod-file-go

Reply:
There was a weird issue where `go mod tidy` was returning a redundant package found in two packages.  I had to "jiggle the handle", so to speak.  This involved a tip to call `go mod tidy -go=1.16 && go mod tidy -go=1.17` and then `go mod tidy` would be successful.  This probably created the additional groups.

Reply:
I manually regrouped to the usual 2 and `go mod tidy` didn't seem to mind.